### PR TITLE
updates from Cactus and Hera for v0.7.7

### PR DESCRIPTION
**Pull request by hu5970**
1) update compilingsetups for Hera. Using spack-stack 1.5.0.
2) change recenter vaiable list to 19.
3) turn off GSI code start run for Hera/Jet.
4) change fit files name from rrfs_a to rrfs.

Tested on Cactus for v0.7.7 and Hera for v0.7.7

### CODE REVIEW

File: scripts/exrrfs_run_analysis.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_run_analysis.sh:1]: With this change, isn't the comment valid for non-WCOSS2 runs as well?

File: scripts/exrrfs_run_recenter.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_run_recenter.sh:1]: I feel like this list keeps getting changed - are we confident in the variable list?

File: ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3
Comment (MatthewPyle-NOAA) [inline ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3:1]: One comment - we don't need to use the quilted restart functionality within the DA system if we don't want to.  I don't think any benefit is gained unless the forecast extends beyond the time the restart file is written.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
A few comments and questions - most important might be trying to clarify if we want to be using the quilted restart functionality with the EnKF forecasts.  Seems like it is impacting the required resources, and shouldn't be necessary.

File: scripts/exrrfs_run_analysis.sh
Comment (hu5970) [reply scripts/exrrfs_run_analysis.sh:1]: This is a but fix. the CALC_ENSMEAN_TN is using the same executable as recentering. So it should use the same module as recentering.

File: scripts/exrrfs_run_recenter.sh
Comment (hu5970) [reply scripts/exrrfs_run_recenter.sh:1]: This change is request from Shun based on v0.7.5 on dogwood. The rrfs should be official name for the system.

File: ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3
Comment (hu5970) [reply ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3:1]: Agree. Seems the quilted restart is turned on automatically. It does not impact anything so far.

Comment (guoqing-noaa) [reply ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3:1]: @hu5970 we may turn off quilting in config_defaults.sh
Currently it is set as `QUILTING="TRUE"`

Comment (MatthewPyle-NOAA) [reply ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3:1]: @guoqing-noaa I think that is for history output (which we do want quilted).  Guessing we need a fresh config option to handle quilted restarts.  @hu5970 if you want to wait to clean up this issue later, I'll approve so we can move on with the multi-phys ensemble.

Comment (hu5970) [reply ush/sample_configs/RRFS_A/config.sh_rrfs_a_enkf_n3:1]: We need to add QUILTING_RESTART="FALSE" to model configure. But I would like to clean up this issue later.

**Review by MatthewPyle-NOAA — APPROVED** [review]
Approving to keep v0.7.7 going - will clean up quilted restart issue later.

**Comment by hu5970** [comment]
@MatthewPyle-NOAA Yes, Please squash merge it.

**Pull request merged**